pyanaconda/ui/gui/spokes/software_selection.py

```diff
@@ -36,6 +36,8 @@
 from pyanaconda.core.async_utils import async_action_wait
 from pyanaconda.ui.categories.software import SoftwareCategory
 
+from pyanaconda.modules.common.constants.services import SUBSCRIPTION, PAYLOAD
```

Comment:
I don't see any changes in TUI. Does the installation in the text mode with a subscription in a kickstart file work as expected? What happens if the subscription fails?

Reply:
Good point, I'll check. Even with no Subscription spoke, we might need to tune up the Source spoke status for partial kickstart installations.

Reply:
So a simple TUI based kickstart installation from CDN seems to work fine.